Eastern-Research-Group/mywaterway
# Feature/315 surrounding monitoring stations layer

## Related Issues:
* https://jira.epa.gov/browse/HMW-315

## Main Changes:
* Added a surrounding past water conditions widget.
  * I went ahead and added this to the community, tribal, and water body report pages. We can show EPA and remove it from those other pages if necessary.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview
2. Turn on the surrounding past water conditions widget
3. Verify it works
4. Navigate to http://localhost:3000/waterbody-report/DOEE/DCANA00E_01/2020
5. Repeat steps 2 and 3.
6. Navigate to http://localhost:3000/tribe/DELAWARENATION
7. Repeat steps 2 and 3.
